### ISSUE TITLE
Build for Windows

### ISSUE THREAD

**Issue by ryfx** [tamaskenez/mapbox-gl-native-builder#1]
Is this build for windows?

**Comment by tamaskenez** [comment]
Yes, the main purpose is to make it build on windows. I also want to make it possible to build the other platforms with CMake.
I expect to finish the Window part next week.

**Comment by tamaskenez** [comment]
My actual fork of mbgl-native is [here](https://github.com/tamaskenez/mapbox-gl-native-cmake). This repo is only a temporary workspace-project to manage the dependencies. It will be replaced by a more elaborate CMake-based package manager solution.

**Comment by ryfx** [comment]
That's great. I want to do this too, but I don't have much time.. Waiting for your good news.

**Comment by tamaskenez** [comment]
Status update: I've finished the first phase. You can build it using this repo and see the instructions in README.md.
You will need MSVC 2015 RC because of the C++11/14 features they use. Even for that I had to fix a thing or two. Now it builds but

- In Debug mode there's an iterator-not-derefenceble assertion
- In Release mode it does show the map and seems working but many labels are missing or rotated incorrectly

I don't know if it's a compiler bug (which would be no surprise for an RC) or something I introduced during porting or some other quirk which works only with gcc/clang.

If I'll have the time I debug it or I wait until July 20 for VS 2015 RTM.

**Comment by tamaskenez** [comment]
@ryfx, it does work with VS2015RTM, however only in Release mode, but all the labels are correct.
In debug mode it crashes without stack, I'll debug it.

**Comment by ryfx** [comment]
I'll install the VS2015RTM,and to build the source code ,Thanks.I'll tell you the result.
